app/main/helpers/search_helpers.py

```diff
@@ -121,6 +121,20 @@ def replace_g5_search_dots(keywords_query):
     )
 
 
+def get_filter_value_from_question_option(option):
+    """
+    Processes `option` into a search term value suitable for the Search API.
+
+    The Search API does not allow commas in filter values, because they are used
+    to separate the terms in an 'OR' search - see `group_request_filters`. They
+    are removed from the field values by the indexer by the Search API itself,
+    see `dm-search-api:app.main.services.conversions.strip_and_lowercase`.
+
+    :return: normalized string value
+    """
+    return (option.get('value') or option.get('label', '')).lower().replace(',', '')
```

Comment:
Yeah, I do kinda agree, this is a slight mess, because they don't quite _need_ to perform the same operation - that regex removes all the spaces, which we don't actually require here. The regex _does already_ get run over the values once they are posted for a search into the Search API, but before we get to that stage we have the delimiter clash.

That said, stripping the spaces before we sent the query wouldn't actually do any harm. Ho hum.